br/pkg/backup/client_test.go

```diff
@@ -826,6 +827,15 @@ func TestObserveStoreChangesAsync(t *testing.T) {
 	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/br/pkg/backup/backup-store-change-tick"))
 }
 
+func genSubRanges(req *backuppb.BackupRequest, count int) {
```

Comment:
if it's referred by only one caller, do we need to make it a standalone internal method?

Reply:
It is called by `TestSplitBackupReqRanges` and `TestSplitBackupReqRanges2`.